### DISCUSSION TITLE
vue表格组件可调节行列宽高

### DISCUSSION THREAD

**Discussion by wangdi190** [antvis/X6#3616] [Q&A]
vue自定义一个可设置行列数的表格，拖拽的画布后，去拖拽表格的行列宽，高。
选中此表格cell时，拖拽里面的单元格，导致cell会跟随transform组件移动。

**Comment by lloydzhou** [comment]
这个是没有处理好事件冒泡的逻辑导致的，我昨天回复了一个相似的问题。

https://github.com/antvis/X6/discussions/3613#discussioncomment-5975326

**Comment by lloydzhou** [comment]
你这边，其实可以在这个table最外层的dom元素上监听一下mousedown和mouseup，阻止冒泡应该就可以了。这样在表格内部进行操作的时候，事件不会冒泡到节点那一层。
